lokeshsingh3721/ClassVault
# fix: fixes the avatar logo in navbar

#what does this PR do 
This PR fixes the avatar logo in the navbar #1 

Fixes #1

![Screenshot (515)](https://github.com/lokeshsingh3721/ClassVault/assets/145474757/96ef4146-e495-49ab-b5b4-cf56a7c8c1d0)

##Type of changes
- Bug Fix

##how should this  be tested
[]- Go to Homepage 
[]- then signup with your account
[]- then See you right corner the avatar logo

## Mandatory task
-[] Make sure  you have self-reviewed the code
